### PR TITLE
Feat/airgrab integration tests

### LINKED ISSUE #3: Add integration tests for Airgrab contract
**Description**

We should add integration tests for the Airgrab contract as defined in the test plan. This test is slightly different in that we want to test with the Fractal signer to verify that our message is in the correct format. To do this we need to confirm the address of the signer. 

There is already an existing branch with some code added as a starting point here:
https://github.com/mento-protocol/governance-tests/tree/test/airgrab-contract

[Airgrab Tests](https://www.notion.so/mentolabs/Governance-Test-Plan-4c38412bebe74095b025b26d2dc34ac0?pvs=4#9c25d24d38c44c88af553e9cfc522c03)

Accounts listed here can be used for claiming: https://www.notion.so/mentolabs/End-to-End-Testing-2cd457e6228645f4bdc77a278b369ebd

**Acceptance Criteria** 

- [x] **Scenario 1: Verify integration tests for Airgrab contract**
  - Verify that the Airgrab contract integration tests are added as defined in the test plan.
  - Verify that the integration tests cover all the necessary scenarios mentioned in the test plan.
  - Verify that the integration tests are written in a way that accurately tests the functionality of the Airgrab contract.

- [x] **Scenario 2: Test coverage for Airgrab contract**
  - Verify that the integration tests cover all the important functionalities of the Airgrab contract.
  - Verify that the integration tests cover both positive and negative scenarios.
  - Verify that the integration tests cover edge cases and handle exceptions appropriately.

- [x] **Scenario 3: Test execution**
  - Verify that the integration tests for the Airgrab contract can be executed successfully.
  - Verify that the test environment is properly set up to execute the integration tests.
  - Verify that the integration tests produce accurate and reliable results.

### PR DESCRIPTION
**Pull request by baroooo**
### Description

Integration tests for the Airgrab contract as defined in the test plan. 

CI fails because of outdated Emission and Locking integration tests (#359 #360)

### Other changes

- Uses env variables to support running on both testnet and mainnet when we deploy there
- Moved tester PK to secrets

### Tested

image

### Related issues

- Fixes #3

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -33,3 +33,9 @@ jobs:
 
       - name: Start node & test
         run: pnpm test:alfajores
+        env:
+          FRACTAL_CLIENT_ID: 'I28eze1-UTsP66nYyBDgiRrGCA7Oe6cRFkl7Gk0Gr1A'
+          MERKLE_PROOF: ${{secrets.MERKLE_PROOF}}
```
Comment (baroooo) [inline .github/workflows/main.yml:38]: this is not a secret, but did not want to disclose it anyways

File: test/Airgrab.test.ts [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+          governanceAddresses.TimelockController,
+        );
+        await mentoToken.connect(governance).unpause();
+      }
+
+      const remainingBalance = await mentoToken.balanceOf(
+        governanceAddresses.Airgrab,
+      );
+      await expect(
+        airgrab.connect(userSigner).drain(governanceAddresses.MentoToken),
```
Comment (nvtaveras) [inline test/Airgrab.test.ts:343]: not like it's a big difference but maybe add a claim before to make it a bit more realistic?

File: test/Airgrab.test.ts [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+            process.env.MERKLE_PROOF!.split(','),
+            proof,
+            validUntil,
+            approvedAt,
+            fractalId,
+          ),
+      ).to.be.revertedWith('Airgrab: KYC no longer valid');
+    });
+
+    it('Should be successfull using a KYCed & eligible account', async function () {
```
Comment (nvtaveras) [inline test/Airgrab.test.ts:208]: I thought we wanted to have an account/test for each blocked country?

**Review by philbow61 — APPROVED** [review]
LGTM 👍

File: test/Airgrab.test.ts (hunk 3 above)
Comment (nvtaveras) [reply test/Airgrab.test.ts:194]: Chatted about this offline, won't do due to the way Fractal sandbox works (TL;DR: users seem to be manually approved/rejected, so doing this won't add much value)

**Review by nvtaveras — APPROVED** [review]
